smoke-test/test_e2e.py

```diff
@@ -1,15 +1,20 @@
 import time
 import urllib
+from contextlib import contextmanager
+from typing import Optional
 
 import pytest
 import requests
-
 from datahub.cli.docker import check_local_docker_containers
 from datahub.ingestion.run.pipeline import Pipeline
-from tests.utils import ingest_file_via_rest
 
-GMS_ENDPOINT = "http://localhost:8080"
-FRONTEND_ENDPOINT = "http://localhost:9002"
+from tests.utils import (
+    get_frontend_url,
+    get_gms_url,
+    get_sleep_info,
+    ingest_file_via_rest,
+)
+
 KAFKA_BROKER = "localhost:9092"
```

Comment:
Kafka broker should also get it's own function that extracts the host via an env var if set.

Reply:
added